tests/test.pipelines.yml

```diff
@@ -1,88 +1,229 @@
+    - path: test-output/batch_1_march/batch_1_march.fl.stderr.log
+    - path: test-output/batch_2_february/batch_2_february.ccs.bam
+    - path: test-output/batch_2_february/batch_2_february.ccs.bam.pbi
+    - path: test-output/batch_2_february/batch_2_february.ccs.report.txt
+    - path: test-output/batch_2_february/batch_2_february.ccs.stderr.log
+    - path: test-output/batch_2_february/batch_2_february.fl.Sample_8_3p--Sample_8_3p-fastqc
+    - path: test-output/batch_2_february/batch_2_february.fl.Sample_9_3p--Sample_9_3p-fastqc
+    - path: test-output/batch_2_february/batch_2_february.fl.json
+    - path: test-output/batch_2_february/batch_2_february.fl.lima.counts
+    - path: test-output/batch_2_february/batch_2_february.fl.lima.report
+    - path: test-output/batch_2_february/batch_2_february.fl.lima.summary
+    - path: test-output/batch_2_february/batch_2_february.fl.stderr.log
+    - path: test-output/batch_3_august/batch_3_august.ccs.bam
+    - path: test-output/batch_3_august/batch_3_august.ccs.bam.pbi
+    - path: test-output/batch_3_august/batch_3_august.ccs.report.txt
+    - path: test-output/batch_3_august/batch_3_august.ccs.stderr.log
+    - path: test-output/batch_3_august/batch_3_august.fl.Sample_2_3p--Sample_2_3p-fastqc
+    - path: test-output/batch_3_august/batch_3_august.fl.Sample_3_3p--Sample_3_3p-fastqc
+    - path: test-output/batch_3_august/batch_3_august.fl.Sample_4_3p--Sample_4_3p-fastqc
+    - path: test-output/batch_3_august/batch_3_august.fl.Sample_7_3p--Sample_7_3p-fastqc
+    - path: test-output/batch_3_august/batch_3_august.fl.Sample_10_3p--Sample_10_3p-fastqc
+    - path: test-output/batch_3_august/batch_3_august.fl.Sample_11_3p--Sample_11_3p-fastqc
+    - path: test-output/batch_3_august/batch_3_august.fl.Sample_12_3p--Sample_12_3p-fastqc
+    - path: test-output/batch_3_august/batch_3_august.fl.Sample_14_3p--Sample_14_3p-fastqc
+    - path: test-output/batch_3_august/batch_3_august.fl.Sample_15_3p--Sample_15_3p-fastqc
+    - path: test-output/batch_3_august/batch_3_august.fl.Sample_17_3p--Sample_17_3p-fastqc
+    - path: test-output/batch_3_august/batch_3_august.fl.Sample_20_3p--Sample_20_3p-fastqc
+    - path: test-output/batch_3_august/batch_3_august.fl.Sample_21_3p--Sample_21_3p-fastqc
+    - path: test-output/batch_3_august/batch_3_august.fl.Sample_24_3p--Sample_24_3p-fastqc
+    - path: test-output/batch_3_august/batch_3_august.fl.Sample_25_3p--Sample_25_3p-fastqc
+    - path: test-output/batch_3_august/batch_3_august.fl.Sample_26_3p--Sample_26_3p-fastqc
+    - path: test-output/batch_3_august/batch_3_august.fl.json
+    - path: test-output/batch_3_august/batch_3_august.fl.lima.counts
+    - path: test-output/batch_3_august/batch_3_august.fl.lima.report
+    - path: test-output/batch_3_august/batch_3_august.fl.lima.summary
+    - path: test-output/batch_3_august/batch_3_august.fl.stderr.log
 
-    - path: test-output/august/august.flnc.E_1_1_3p--E_1_1_3p.bam
```

Comment:
Can you add in a few tests with `should_exist: false` to make sure that no polished/`flnc` files are generated when the input is DNA?

Reply:
I have also added a `should_exist: false` for the fastqc tasks that handles `fl` files which should not occur in the RNA run.